src/classes/ERR_Handler.cls

```diff
@@ -440,19 +447,19 @@ public class ERR_Handler {
         }
         return null;
     }
-    
+
     /*******************************************************************************************************
     * @description Returns the status of the error that happened as a result of a DML operation performed through
     *              the Database class as a string.
     * @param status The status code.
     * @return The error status as a string.
     */
-    private static String getStatusCode(system.StatusCode status) {
+    public static String getStatusCode(system.StatusCode status) {
         if (status == system.Statuscode.REQUIRED_FIELD_MISSING)
-            return 'Required Field Missing';   
+            return System.Label.errErrorTypeRequiredFieldMissing;
```

Comment:
Might as well add the curly brackets if you're modifying this method. Sorry I missed this one before.